examples/chatgpt/gpt_actions_library/gpt_action_snowflake_direct.ipynb

```diff
@@ -312,11 +345,30 @@
    "metadata": {},
    "source": [
     "\n",
-    "* Retrieve your OAuth Client Secret\n",
+    "* Retrieve your OAuth Client Secret"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "yaml"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "select SYSTEM$SHOW_OAUTH_CLIENT_SECRETS('CHATGPT_INTEGRATION');"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
     "\n",
-    "You’ll find the Client Secret in OAUTH_CLIENT_SECRET.\n",
+    "You’ll find the Client Secret in OAUTH_CLIENT_SECRET. Do not include the double quotes when copying the value.\n",
     "\n",
-    "You’re now set to test your action in Postman. Once you get a successful API response there, you can proceed with your GPT Action."
+    "Now is a good time to [test your Snowflake integration in Postman](https://community.snowflake.com/s/article/How-to-configure-postman-for-testing-SQL-API-with-OAuth). If you choose to do so, ensure that your network policy includes the IP of the machine you're using to test."
```

Comment:
would add "assuming you set one up" if the step is optional

Reply:
updated to consider that!